uint8 choice
int16 angle
